src/effects/EffectUI.cpp

```diff
@@ -1158,10 +1138,16 @@ void EffectUIHost::LoadUserPresets()
 
 void EffectUIHost::InitializeRealtime()
 {
-   if (mSupportsRealtime && !mInitialized)
-   {
-      RealtimeEffectManager::Get(*mProject).RealtimeAddEffect(mEffect);
-      
+   if (mSupportsRealtime && !mInitialized) {
+      mpState =
```

Comment:
this part is affected by the same problem we have seen in PR 2500 i.e. mpState will be returned null - this because the code inside AudioIO::AddState was probably cherry-picked from Realtime-effect-manager-rewrite-part3. 
I can not test how effects work until that part will be fixed - so please merge future fixes from Realtime-effect-manager-rewrite-part3 into this and I can try again.